spec/Classes.tex

```diff
@@ -307,12 +370,12 @@ \subsection{Class Values}
 \end{chapelcompopts}
```

Comment:
Could you change Line 358 above to say:
`... is deleted automatically when it is not`